src/WebClient.ts

```diff
@@ -656,6 +651,8 @@ export interface WebAPIHTTPError extends CodedError {
  * Helpers
  */
 
+const defaultFilename = 'Untitled';
```

Comment:
Should we add a note to the docs for this?

Reply:
heh, if we had generated docs for the individual methods at all, then i'd say yes.